static/scripts/ckeditor/ckeditor.js

```diff
@@ -1,18 +1,51 @@
 import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
 import ckeditorConfig from './ckeditor-config';
 
+const url = window.location.pathname;
```

Comment:
Would be better using the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL_API) to avoid the workaround of playing with the array positions of the url.

Reply:
in the static scripts, we have no other means to access the current components
URL API would be server, the homework is on client